crates/bevy_hierarchy/src/query_extension.rs

```diff
@@ -214,7 +214,7 @@ where
     }
 }
 
-impl<'w, 's, D: QueryData, F: QueryFilter> Iterator for DescendantIter<'w, 's, D, F>
+impl<'w, D: QueryData, F: QueryFilter> Iterator for DescendantIter<'w, '_, D, F>
```

Comment:
I really dislike this change too.

Reply:
I don't like this change because there is still a lifetime parameter, so it's weird to switch between eliding and not eliding.